src/Illuminate/Notifications/Channels/MailChannel.php

```diff
@@ -95,13 +96,19 @@ protected function buildView($message)
             return $message->view;
         }
 
-        if (property_exists($message, 'theme') && ! is_null($message->theme)) {
-            $this->markdown->theme($message->theme);
+        /**
+         * We clone the Markdown instance so that setting a theme doesn't
+         * carry over to subsequent calls to build the view.
+         */
+        $markdown = clone $this->markdown;
```

Comment:
I would rather to have an implementation exactly like my pull request, and simply access the instance of markdown (`$this->markdown`) within the closure, to set the theme based on the message, right before call the renders.

Reply:
I have added what I think you requested. If the code changes look good, I can go and fix the tests that are failing due to Mockery expectations.

That said, I respectfully disagree with this approach. Why even bother passing a default theme to the Markdown class if we are always going to manually fetch the markdown theme from config() every time it's used?

Reply:
Maybe this would be better if there was a PendingRender class or something. Maybe an idea for 11.x